source/code/source/playfab/PlayFabPluginManager.cpp

```diff
@@ -61,12 +61,12 @@ namespace PlayFab
         const auto pluginEntry = plugins.find(key);
         if (pluginEntry == plugins.end())
         {
```

Comment:
food for thought, have you considered opportunistically simplifying this method?

the `if end then {insert} else {delete and insert}` is unnecessarily verbose

```
key = ...;
plugins[key] = std::move(plugin);
```

is functionally equivalent and easier to read IMO